DopplerFiles/appsettings.Development.json

```
@@ -12,12 +12,17 @@
   "AwsS3Settings": {
     "AccountInfo": {
       "AccessKey": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis eu.",
-      "SecretAccessKey": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis eu."
+      "SecretAccessKey": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis eu.",
+      "RegionEndpoint": "USEast1"
     },
     "BucketName": "doppler-files-local"
   },
   "ServiceSettings": {
-    "UsersRootFolder": "Users"
+    "UsersRootFolder": "Users",
+    "SuperUserFolder": "Su",
```

I am not sure about the meaning of this folder, but maybe it is a shared one?

```suggestion
    "SuperUserFolder": "shared",
```

---

Thinking it twice, why do you need a special folder for SU? In what scenario will you use it? 

Maybe you can simplify it allowing the SU to upload files to any user folder, but without a special folder for him.

---

Thinking about that... Superuser folder is the default folder for a SU but a SU should upload at any folder.